src/utils/test/validAttr.test.js

```diff
@@ -164,392 +164,392 @@ describe('validAttr', () => {
+  //   expect(validAttr('onCanPlayThroughCapture')).toEqual(true)
+  //   expect(validAttr('onDurationChange')).toEqual(true)
+  //   expect(validAttr('onDurationChangeCapture')).toEqual(true)
+  //   expect(validAttr('onEmptied')).toEqual(true)
+  //   expect(validAttr('onEmptiedCapture')).toEqual(true)
+  //   expect(validAttr('onEncrypted')).toEqual(true)
+  //   expect(validAttr('onEncryptedCapture')).toEqual(true)
+  //   expect(validAttr('onEnded')).toEqual(true)
+  //   expect(validAttr('onEndedCapture')).toEqual(true)
+  //   expect(validAttr('onError')).toEqual(true)
+  //   expect(validAttr('onErrorCapture')).toEqual(true)
+  //   expect(validAttr('onLoadedData')).toEqual(true)
+  //   expect(validAttr('onLoadedDataCapture')).toEqual(true)
+  //   expect(validAttr('onLoadedMetadata')).toEqual(true)
+  //   expect(validAttr('onLoadedMetadataCapture')).toEqual(true)
+  //   expect(validAttr('onLoadStart')).toEqual(true)
+  //   expect(validAttr('onLoadStartCapture')).toEqual(true)
+  //   expect(validAttr('onPause')).toEqual(true)
+  //   expect(validAttr('onPauseCapture')).toEqual(true)
+  //   expect(validAttr('onPlay')).toEqual(true)
+  //   expect(validAttr('onPlayCapture')).toEqual(true)
+  //   expect(validAttr('onPlaying')).toEqual(true)
+  //   expect(validAttr('onPlayingCapture')).toEqual(true)
+  //   expect(validAttr('onProgress')).toEqual(true)
+  //   expect(validAttr('onProgressCapture')).toEqual(true)
+  //   expect(validAttr('onRateChange')).toEqual(true)
+  //   expect(validAttr('onRateChangeCapture')).toEqual(true)
+  //   expect(validAttr('onSeeked')).toEqual(true)
+  //   expect(validAttr('onSeekedCapture')).toEqual(true)
+  //   expect(validAttr('onSeeking')).toEqual(true)
+  //   expect(validAttr('onSeekingCapture')).toEqual(true)
+  //   expect(validAttr('onStalled')).toEqual(true)
+  //   expect(validAttr('onStalledCapture')).toEqual(true)
+  //   expect(validAttr('onSuspend')).toEqual(true)
+  //   expect(validAttr('onSuspendCapture')).toEqual(true)
+  //   expect(validAttr('onTimeUpdate')).toEqual(true)
+  //   expect(validAttr('onTimeUpdateCapture')).toEqual(true)
+  //   expect(validAttr('onVolumeChange')).toEqual(true)
+  //   expect(validAttr('onVolumeChangeCapture')).toEqual(true)
+  //   expect(validAttr('onWaiting')).toEqual(true)
+  //   expect(validAttr('onWaitingCapture')).toEqual(true)
+  //   expect(validAttr('onLoad')).toEqual(true)
+  //   expect(validAttr('onLoadCapture')).toEqual(true)
+  //   expect(validAttr('onAnimationStart')).toEqual(true)
+  //   expect(validAttr('onAnimationStartCapture')).toEqual(true)
+  //   expect(validAttr('onAnimationEnd')).toEqual(true)
+  //   expect(validAttr('onAnimationEndCapture')).toEqual(true)
+  //   expect(validAttr('onAnimationIteration')).toEqual(true)
+  //   expect(validAttr('onAnimationIterationCapture')).toEqual(true)
+  //   expect(validAttr('onTransitionEnd')).toEqual(true)
+  //   expect(validAttr('onTransitionEndCapture')).toEqual(true)
```

Comment:
React 0.14 [does support](https://github.com/facebook/react/blob/0.14-stable/src/renderers/dom/shared/SVGDOMPropertyConfig.js#L24) some of these but it was really fiddly working out which ones so I took the nuclear option for now.